cli/types/index.d.ts

```diff
@@ -4090,15 +4090,6 @@ declare namespace Cypress {
      */
     (chainer: 'not.match', value: string): Chainable<Subject>
 
-    // fallback
-    /**
-     * Create an assertion. Assertions are automatically retried until they pass or time out.
-     * Ctrl+Space will invoke auto-complete in most editors.
-     * @see https://on.cypress.io/should
-     */
-    (chainers: string, value?: any): Chainable<Subject>
-    (chainers: string, value: any, match: any): Chainable<Subject>

```

Comment:
PS: if a user has custom chainers they want to use, they should be able to explicitly add them by extending the `Chainer` `interface`. There is no need for generic `string`/`any` fallbacks.